src/zenml/integrations/deepchecks/materializers/deepchecks_dataset_materializer.py

```diff
@@ -63,6 +64,21 @@ def save(self, df: Dataset) -> None:
         pandas_materializer = PandasMaterializer(self.uri)
         pandas_materializer.save(df.data)
 
+    def save_visualizations(self, df: Dataset) -> Dict[str, VisualizationType]:
+        """Saves visualizations for the given Deepchecks dataset.
+
+        Args:
+            df: The Deepchecks dataset to save visualizations for.
```

Comment:
somehow the naming feels a bit confusing. why `df` for dataset?

Reply:
That's how the existing methods called it. But you're right, `df` is confusing, renamed it to `dataset` everywhere.